.ruby-version

```diff
@@ -1 +1 @@
-2.0.0
+2.1.0
```

Comment:
Blz... tive que trocar aqui na minha máquina para rodar o projeto, mas
esqueci de voltar ao original antes do push. Não esquecerei nas próximas.😀

Em 11 de maio de 2016 10:00, Leandro Camargo <notifications@github.com>
escreveu:

> In .ruby-version
> <https://github.com/gorails/gorails/pull/93#discussion_r62842854>:
>
> > @@ -1 +1 @@
> > -2.0.0
> > +2.1.0
>
> Por enquanto evite de alterar a versão do .ruby-version, pois isso quebra
> algumas coisas em produção. O @jcottobboni
> <https://github.com/jcottobboni> sabe te explicar melhor essa história.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gorails/gorails/pull/93/files/8c8afeedc883b72b63cd606249759da07457b2c5#r62842854>
>